app/controllers/programming_groups_controller.rb

```diff
@@ -4,7 +4,18 @@ class ProgrammingGroupsController < ApplicationController
   include CommonBehavior
   include LtiHelper
 
-  before_action :set_exercise_and_authorize
+  before_action :set_exercise_and_authorize, only: %i[new create]
+  before_action :set_programming_group, only: %i[show edit update destroy]
```

Comment:
We have a variable for that:

```suggestion
  before_action :set_programming_group, only: MEMBER_ACTIONS
```